Lint检查失败: 新增代码存在 3 个 ESLint 错误

## 测试输出错误信息

```
/Users/hs3180/clawd/disclaude/src/channels/feishu-channel-passive-mode.test.ts
  526:64  error  Async arrow function has no 'await' expression  require-await

/Users/hs3180/clawd/disclaude/src/nodes/commands/builtin-commands.ts
  272:3   error  Async method 'execute' has no 'await' expression  require-await
  287:16  error  Strings must use singlequote                      quotes

✖ 68 problems (3 errors, 65 warnings)
```

## 根本原因分析

### 错误 1 & 2: `require-await` 违规

两个 `async` 函数没有使用 `await` 表达式：

1. **feishu-channel-passive-mode.test.ts:526** - 测试用例声明为 `async` 但内部无 `await`
   ```typescript
   it('should correctly report passive mode status', async () => {
     // 函数内只有同步的 expect 语句，不需要 async
   ```

2. **builtin-commands.ts:272** - `PassiveCommand.execute` 方法声明为 `async` 但无 `await`
   ```typescript
   async execute(context: CommandContext): Promise<CommandResult> {
     // 方法内只有同步操作，不需要 async
   ```

### 错误 3: `quotes` 违规

**builtin-commands.ts:287** - 模板字符串中使用了不正确的引号字符：
```typescript
message: `🔄 **被动模式设置中...**`,
//                    ↑ 这里的引号字符不符合 ESLint 要求
```

## 触发提交

以下提交引入了这些问题：
- `fb435aa` Merge pull request #543 from hs3180/feat/issue-511-passive-mode-control
- `f9658b2` feat(group): add passive mode control command for group chats (Issue #511)

## 建议修复

1. 移除不必要的 `async` 关键字，或
2. 如果预期将来会有异步操作，添加 `// eslint-disable-next-line require-await` 注释
3. 检查并修复模板字符串中的引号字符

## 相关文件

- src/channels/feishu-channel-passive-mode.test.ts
- src/nodes/commands/builtin-commands.ts

--- a/src/channels/feishu-channel-passive-mode.test.ts
+++ b/src/channels/feishu-channel-passive-mode.test.ts
@@ -523,7 +523,7 @@ describe('FeishuChannel - Group Chat Passive Mode (Issue #460)', () => {
       expect(messageHandler).toHaveBeenCalledTimes(1); // Still 1, not incremented
     });
 
-    it('should correctly report passive mode status', async () => {
+    it('should correctly report passive mode status', () => {
       // Initially passive mode is enabled (not disabled)
       expect(channel.isPassiveModeDisabled('oc_test_group')).toBe(false);
 

--- a/src/nodes/commands/builtin-commands.ts
+++ b/src/nodes/commands/builtin-commands.ts
@@ -269,7 +269,7 @@ export class PassiveCommand implements Command {
   readonly description = '群聊被动模式开关';
   readonly usage = 'passive [on|off|status]';
 
-  async execute(context: CommandContext): Promise<CommandResult> {
+  execute(context: CommandContext): CommandResult {
     // Default to status if no args
     const subCommand = context.args[0]?.toLowerCase() || 'status';
 
@@ -284,7 +284,7 @@ export class PassiveCommand implements Command {
     // Actual implementation is handled by PrimaryNode/CommunicationNode
     return {
       success: true,
-      message: `🔄 **被动模式设置中...**`,
+      message: '🔄 **被动模式设置中...**',
     };
   }
 }

